client/reader/controller.js

```diff
@@ -176,24 +171,20 @@ const exported = {
 	},
 
 	feedListing( context, next ) {
-		const basePath = '/read/feeds/:feed_id',
-			fullAnalyticsPageTitle = analyticsPageTitle + ' > Feed > ' + context.params.feed_id,
-			feedStore = feedStreamFactory( 'feed:' + context.params.feed_id ),
-			mcKey = 'blog';
-
-		ensureStoreLoading( feedStore, context );
+		const basePath = '/read/feeds/:feed_id';
+		const fullAnalyticsPageTitle = analyticsPageTitle + ' > Feed > ' + context.params.feed_id;
+		const mcKey = 'blog';
+		const feedId = context.params.feed_id;
```

Comment:
moving this higher would make the fullAnalyticsPageTitle simpler

Reply:
will do